clarifai_grpc/channel/http_client.py

```diff
@@ -90,8 +90,10 @@ def _mangle_base64_values(self, params):  # type: (dict) -> dict
 
     def _mangle_base64_values_in_inputs(self, params):  # type: (dict) -> dict
         params_copy = copy.deepcopy(params)
-        for data in params_copy["inputs"]:
-            data = data["data"]
+        for input in params_copy["inputs"]:
+            if "data" not in input:
+                continue
```

Comment:
If the "data" is not set, we should check the "data" object in "outputs" instead:
`if "data" not in input:
    if "outputs" not in input:
        continue
    else:
        data = input["outputs"]["data"]
else:
    data = input["data"]
`

Reply:
This is only for logging not response. All good.